beacon-chain/rpc/apimiddleware/structs.go

```diff
@@ -206,28 +206,29 @@ type aggregateAttestationResponseJson struct {
 	Data *attestationJson `json:"data"`
 }
 
+// submitAggregateAndProofsRequestJson is used in /validator/aggregate_and_proofs API endpoint.
+type submitAggregateAndProofsRequestJson struct {
+	Data []*signedAggregateAttestationAndProofJson `json:"data"`
+}
+
 //----------------
 // Reusable types.
 //----------------
 
-// checkpointJson is a JSON representation of a checkpoint.
```

Comment:
I removed these redundant comments. The name of the type carries sufficient information about the intent.